src/services/api.ts

```
@@ -175,6 +177,7 @@ const shouldRetry = (error: AxiosError): boolean => {
 const handleHttpError = (status: number, appStore: ReturnType<typeof useAppStore>): void => {
   switch (status) {
     case 401:
+      const { login } = useAuth()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)